mysql-client/Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM ubuntu
-MAINTAINER OpenEBS
-RUN apt-get update
-RUN apt-get install -y mysql-client timelimit jq moreutils 
+FROM alpine
+
+LABEL maintainer="OpenEBS"
+
+RUN apk add --no-cache mysql-client && apk add --no-cache jq && apk add --no-cache bash
```

Comment:
@bilal-mk I think that adding **--no-cache** bash will not support all the bash command. What we can do is either installing **moreutils** package or we can use the base image as ubuntu only. Please accomodate this change it will highy appreciated.

Reply:
Added moreutils package

Reply:
@shashank855 moerutils is necessitated on account of usage of `ts` in the script. --no-cache shouldn't matter IMO.